### ISSUE TITLE
Consider using OSM/Nominatim instead of Google Maps

### ISSUE THREAD

**Issue by animaux** [nthmedia/entry-gps-coordinates#17]
Have you ever considered using OSM/Nominatim instead of Google services? Would be nice to have a version independent of Googles assets and services.

Example: https://github.com/animaux/maplocationfield

**Comment by paul-frost** [comment]
I would love an OpenStreetMap version, I'm trying to get away from Google.
I love the simplicity of this field, I just want to capture the lat/lng numbers. I can then choose my own MapBox,Leaflet or diy method of showing maps on the site.
I build sites for orienteering clubs, and events/races are often held in remote locations where there are no postcodes or addresses.

**Comment by niektenhoopen** [comment]
@paul-frost @animaux I agree and would like to support OSM. I just have not had the time to look into it yet and I don't think that will happen before Christmas to be honest. If you have a chance to create a pull request, feel free to do so. I am happy to review and merge :-)
